extra-multimedia/vapoursynth/autobuild/prepare

```diff
@@ -1,4 +1,5 @@
-export CXXFLAGS="${CXXFLAGS/-fdeclone-ctor-dtor/}"
+#export CXXFLAGS="${CXXFLAGS/-fdeclone-ctor-dtor/}"
```

Comment:
Remove all lines if you didn't need it, this is Git, don't comment out code.

Reply:
Resolved in 14843fe.